app/assets/stylesheets/custom.scss

```diff
@@ -40,25 +41,27 @@ body {
   background-color: $dark-blue2;
 }
 
+.container {
+  width: 95%;
+}
 .main-body {
-  padding-top: 10px;
+  padding-top: 30px;
   text-align: center;
+  width: 80%;
   // // background: blue;
   // width: 110%;
   // padding-right: -15px;
 }
 
+.row {
+  margin: 30px 0 50px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `30px 0 50px` instead of `30px 0 50px 0`